src/index.js

```diff
@@ -5,6 +5,7 @@ import { Provider } from 'react-redux';
 import store from './redux/configureStore';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+// import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
 
```

Comment:
- [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.